doc/man3/flux_kvs_txn_create.adoc

```diff
@@ -31,6 +31,10 @@ SYNOPSIS
  int flux_kvs_txn_symlink (flux_kvs_txn_t *txn, int flags,
                            const char *key, const char *target);
 
+ int flux_kvs_txn_put_raw (flux_kvs_txn_t *txn, int flags,
+                           const char *key, void *data, int len);
+
```

Comment:
```const void *data```?

Reply:
I think you might be looking at an older version there?  This is already done.

I probably shouldn't have forced pushed a squashed version back there.

Reply:
still shows up ```void *``` to me.  This is the adoc, not the header file.